src/untangled/client/impl/application.cljc

```diff
@@ -205,7 +205,7 @@
   To resolve the issue, we def an atom pointing to the reconciler that the send method will deref each time it is
   called. This allows us to define the reconciler with a send method that, at the time of initialization, has an app
   that points to a nil reconciler. By the end of this function, the app's reconciler reference has been properly set."
-  [{:keys [send-queues mutation-merge] :as app} initial-state parser {:keys [pathopt migrate shared] :or {pathopt true migrate nil shared nil}}]
+  [{:keys [send-queues mutation-merge] :as app} initial-state parser {:keys [migrate] :as reconciler-options}]
   (let [rec-atom                  (atom nil)
         remotes                   (keys send-queues)
         tempid-migrate            (fn [pure _ tempids _]
```

Comment:
This should move to a top-level public function so ppl can compose it in if they want to plug into migrate. Also that would allow a test to say it is the default.